Formula/ucloud.rb

```diff
@@ -0,0 +1,21 @@
+class Ucloud < Formula
+  desc "The official tool to managment your ucloud services"
+  homepage "https://www.ucloud.cn"
+  url "ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.1.tar.gz"
+  sha256 "d10ea007c099280c011a53a074232e143e79576fbcb84c6198e7de23e95005ef"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/ucloud/ucloud-cli"
+    dir.install buildpath.children
+    cd dir do
+      system "go", "build", "-o", bin/"ucloud"
+    end
+  end
+
+  test do
+    system "#{bin}/ucloud", "version"
```

Comment:
Please try to write a test that exercises a deeper level of functionality. Thanks!